.github/workflows/post_deploy_asset_check.yml

```diff
@@ -19,15 +21,25 @@ on:
         required: true
 
 jobs:
-  asset-check:
+  notify-of-checks-start:
     runs-on: ubuntu-latest
     steps:
-      - name: Check out code
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+      - name: Notify via Slack that tests are starting
+        uses: ./.github/actions/slack
         with:
-          ref: ${{ inputs.git_sha }}
-          fetch-depth: 0
+          channel_id: ${{ env.SLACK_CHANNEL_ID }}
+          env_name: test
+          fetch-depth: 1
```

Comment:
```suggestion
```

Reply:
_(not familiar with the slack action code, but assuming it doesn't need to access any paths other than its own code so it could be the only folder checked out instead of the whole tree, but didn't really study up what it ends up running.)_